plugin/src/sbt-test/paradox/generated-source/build.sbt

```diff
@@ -0,0 +1,9 @@
+lazy val docs = project
+  .in(file("."))
+  .enablePlugins(ParadoxPlugin)
+  .settings(
+    name := "Paradox Source Generation Test",
+    paradoxProperties += ("canonical.base_url" -> "https://example.com/doc/"),
+    paradoxRoots := List("index.html"),
```

Comment:
I'm assuming `paradoxProperties` and `paradoxRoots` are not needed for this test.

Reply:
Also, add a `paradoxTheme := None` setting to remove all default theme stuff from the expected output.

Add an empty template file `src/main/paradox/_template/page.st` with:

```
$page.content$
```

And then you'll just have the minimum expected files.

Reply:
Done! Cheers